ShellPkg/Application/Shell/ShellProtocol.c

```diff
@@ -1800,8 +1822,10 @@ EfiShellExecute (
   OUT EFI_STATUS  *StatusCode OPTIONAL
   )
 {
-  EFI_STATUS                Status;
-  CHAR16                    *Temp;
+  EFI_STATUS  Status;
+  // MU_CHANGE [START] - CodeQL change
+  CHAR16  *Temp = NULL;

```

Comment:
nit: spacing is weird here now, maybe put the MU_CHANGE as a comment to the right of Temp so that all vars can be aligned spacing wise.